### DISCUSSION TITLE
How to Fine-Tune or Create my Own Model from Existing or Scratch ?

### DISCUSSION THREAD

**Discussion by codezeros** [lllyasviel/Fooocus#2928] [Q&A] [answered]
Hi  i would also like to fine-tune and train my own LLMs is there any guidelines for that with either from any existing model like llama or any other model or from scratch and also for stable diffusion models with fully base model or as loras on locally using GPU? with own images and datas in .safetensors or onnx or .pickle tensors like file formats

**Comment by IPv6** [comment, marked as answer]
Fooocus designed purely for inference, there is zero facilities to train anything. Besides, training stable diffusion and LLM are vastly different processes. You need to install Automatic WebUI or similar and google for tutorials, there is a lot of them. Fooocus will not help in this

**Comment by codezeros** [comment]
> Fooocus designed purely for inference, there is zero facilities to train anything. Besides, training stable diffusion and LLM are vastly different processes. You need to install Automatic WebUI or similar and google for tutorials, there is a lot of them. Fooocus will not help in this

yeah is there any collab or any other material to learn from ? also how does big tech company use large amount of gpu to train and create thier own models?  like in civitai and hugging face???

**Reply by IPv6** [reply]
i suppouse there is a lot of tutorials and stuff - you`d better ask such questions on reddit/etc since it does not relate to Fooocus.
